### PR TITLE
First release :tada:

### PR DESCRIPTION
**Pull request by PericlesTheo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,131 @@
...
+import CreditDigital from "creditdigital-widget";
+
+export default class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.creditDigital = new CreditDigital({ skipCSSInjection: true });
+  }
+
+  render() {
+    return <p>The monthly price will be { this.productListingMonthlyRate(props.totalAmount) }</p>;
```
Comment (isaacseymour) [inline README.md]: `this.creditDigital.productListingMonthlyRate`

File: README.md [hunk 2]
```diff
@@ -1 +1,131 @@
...
+- `callback_url` (optional): A webhook url to keep you informed on the status of a particular application.
+- `invoice_number` (optional): An invoice number to be attached to the CreditDigital application for your reference.
+- `invoice_description` (optional): An invoice description to be attached to the CreditDigital application for your reference.
+
+### Minimum Amount
+CreditDigital by default imposes a £250 minimum amount. If the total amount of a product or checkout is less than the minimum amount the checkout button will not appear and the product listing box will inform the user about the minimum.
+
+You can change the minimumAmount to fit your requirements as long as it's more than the CreditDigital imposed one:
+```javascript
+var creditDigital = new Creditdigital();
```
Comment (isaacseymour) [inline README.md]:
```suggestion
var creditDigital = new CreditDigital();
```

File: src/index.ts [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+  public setMinimumAmount(minimumAmount: number) {
+    if (minimumAmount < MINIMUM_AMOUNT_BY_CREDITDIGITAL) {
+      console.warn(`Minimum amount cannot be lower than £${MINIMUM_AMOUNT_BY_CREDITDIGITAL}`);
+    } else {
+      this.minimumAmount = minimumAmount;
+    }
+  }
+
+  public productListingMonthlyRate(productPrice: number): string {
```
Comment (PericlesTheo) [inline src/index.ts]: should check the `minimumAmount` as well

File: src/index.ts [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+    if (data.cash_price < this.minimumAmount) {
+      return;
+    }
+    if (!node) {
+      return;
+    }
+
+    node.innerHTML = `<div>
+        <a style="color:black;cursor:pointer;" id="credit-digital-checkout">
+          <img src="https://widget.creditdigital.co.uk/paywithcd.png"https://widget.creditdigital.co.uk/paywithcd.png" />
```
Comment (isaacseymour) [inline src/index.ts]: double paste in here

File: README.md [hunk 5]
```diff
@@ -1 +1,135 @@
...
+
+### CSS rules
+CreditDigital by default, will inject default styles for the product listing box. You can opt-out via:
+```javascript
+creditDigital = new CreditDigital({ skipCSSInjection: true })
+```
+
+*Note: You might want to opt-out in the checkout page if you are not planning to display the product listing container.*
+
+If you prefer to override some or all of the styles to make the resulting HTMN match the look and fill of your website, these are the classes you can override:
```
Comment (isaacseymour) [inline README.md]:
```suggestion
If you prefer to override some or all of the styles to make the resulting HTML match the look and fill of your website, these are the classes you can override:
```

File: examples/checkout.html [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+    // from various other changes in the UI.
+    // The example we will use is increasing the quantity of the product we want to buy
+
+    var addToCardTotalEle = document.getElementById("total-for-cart");
+
+    document.getElementById("quantity").addEventListener("change", function (e) {
+      var quantitySelected = e.target.value;
+      var totalCost = price * quantitySelected;
+
+      creditDigital.productListingHtml(targetNode, price * quantitySelected);
```
Comment (isaacseymour) [inline examples/checkout.html]: There's no quantity field on this page ✂️

**Pull request merged**